lib/util.js

```diff
@@ -64,16 +68,28 @@ schemaFaker.option({
 * Safe wrapper for schemaFaker that resolves references and
 * removes things that might make schemaFaker crash
 * @param {*} oldSchema the schema to fake
+* @param {string} resolveTo tells to use example values or fake schema
 * @param {string} bodyTypeOption tells that the schema object is of request or response
 * @param {*} components list of predefined components (with schemas)
 * @param {string} schemaFormat default or xml
 * @param {string} indentCharacter char for 1 unit of indentation
 * @returns {object} fakedObject
 */
-function safeSchemaFaker(oldSchema, bodyTypeOption, components, schemaFormat, indentCharacter) {
+function safeSchemaFaker(oldSchema, resolveTo, bodyTypeOption, components, schemaFormat, indentCharacter) {
   var prop,
     schema = deref.resolveRefs(oldSchema, bodyTypeOption, components);
 
+  if (resolveTo === 'schema') {
```

Comment:
Also, check the copy for the jsdoc descriptions. `tells to` is incorrect. Every enum option should:
1. Describe what it does
2. Explain each of the options
3. Mention the default

`@param {string} resolveTo The desired JSON-generation mechanism (schema: prefer using the JSONschema to generate a fake object, example: use specified examples as-is). Default: schema`

Even the desc for bodyTypeOption doesn't read well. `Specifies whether the schema being faked is from a request or response.`. Also, explain how this affects the output.